deploy.go

```diff
@@ -194,6 +196,28 @@ func RunDeploy(c *cli.Context) error {
 	return nil
 }
 
+func displayNewCommits(owner string, repo string, c *cli.Context, client *github.Client) {
+	ref := Ref(c.String("ref"), git.Head)
+
+	opt := &github.DeploymentsListOptions{
+		Environment: c.String("env"),
+	}
+
+	deployments, _, error := client.Repositories.ListDeployments(owner, repo, opt)
```

Comment:
error should probably be err, also you don't seem to handle the case where err != nil